server.js

```diff
@@ -0,0 +1,39 @@
+var express = require("express")
```

Comment:
Tem algum motivo pra usar var e não const?

Reply:
Utilizei a documentação do própio Graphql, mas se for for viável realizarei a alteração.